docs/tutorials/hyp3_insar_stack_for_ts_analysis.ipynb

```diff
@@ -375,6 +386,8 @@
     "mintpy.load.incAngleFile     = {data_dir}/*/*_lv_theta_clipped.tif\n",
     "mintpy.load.azAngleFile      = {data_dir}/*/*_lv_phi_clipped.tif\n",
     "mintpy.load.waterMaskFile    = {data_dir}/*/*_water_mask_clipped.tif\n",
+    "##----------troposphere correction\n",
```

Comment:
I think the parameter name is self-explanatory without this comment, so I recommend dropping the comment